SCYLLA-VERSION-GEN

```diff
@@ -58,6 +64,10 @@ if [ -z "$OUTPUT_DIR" ]; then
 	OUTPUT_DIR="$SCRIPT_DIR/build"
 fi
 
+if [ -z "$DATE" ]; then
+  DATE=$(date --utc +%Y%m%d)
```

Comment:
You have to clear DATE before parsing options so it isn't picked from the environment.

Reply:
Done